client/src/components/content/ModelerSidebar/PropertiesPanel/PropertiesPanel.jsx

```diff
@@ -23,12 +23,13 @@ const PropertiesPanel = ({
   selectedActivity,
   taskSelectionUpdated,
   disableTaskSelection,
+  robotId,
```

Comment:
linter warning

Reply:
✅

Reply:
FYI: The linter warning says that `robotId` is missing in props validation, so please add
```jsx
  robotId: PropTypes.string.isRequired,
```
in line 67